### PR TITLE
Bring up-to-date with latest anbox

### LINKED ISSUE #47: VNC
Once I disconnect from VNC. I can't connect anymore! 
I have to remove and recreate the container to get 1 vnc connection

### LINKED ISSUE #48: Different major and minor device numbers compared to anbox-modules
anbox-modules installs /dev/ashmem and /dev/binder with the following major and minor numbers:
$ ls -al /dev/ashmem 
crw------- 1 root root 10, 55 Aug 28 09:41 /dev/ashmem
$ ls -al /dev/binder
crw------- 1 root root 511, 0 Aug 28 09:41 /dev/binder
This is according to "Install Instructions" on https://github.com/anbox/anbox-modules

aind creates both devices in the following way (https://github.com/aind-containers/aind/blob/master/src/anbox-container-manager-pre.sh):
mknod /dev/ashmem c 10 52
mknod /dev/binder c 10 59

I propose to change the major, minor device numbers in aind, so that they are the same as in anbox-modules.

### PR DESCRIPTION
**Pull request by Niek**
This PR bring aind up-to-date with the latest anbox. Supersedes https://github.com/aind-containers/aind/pull/49

Added also a fix for x11vnc quitting after a disconnect (fix https://github.com/aind-containers/aind/issues/47)

Added also a fix for incorrect devices versions (fix #48)

### CODE REVIEW

**Review by AkihiroSuda — APPROVED** [review]
Thanks

File: Dockerfile [hunk 1]
```diff
@@ -48,20 +49,18 @@ RUN apt-get update && \
   pkg-config \
   protobuf-compiler \
   python2
-RUN git clone https://github.com/anbox/anbox /anbox
+RUN git clone --recursive https://github.com/anbox/anbox /anbox
 WORKDIR /anbox
 ARG ANBOX_COMMIT
-RUN git pull && git checkout ${ANBOX_COMMIT}
+RUN git pull && git checkout ${ANBOX_COMMIT} && git submodule update --recursive
```
Comment (SuperSandro2000) [inline Dockerfile:55]: You should probably add ``--init`` to the submodule update.

Comment (Niek) [reply Dockerfile:55]: Why is that? As far as I'm aware that is only needed if you want to update the `.git/config` file.

Comment (SuperSandro2000) [reply Dockerfile:55]:
```
If the submodule is not yet initialized, and you just want to use the setting as stored in .gitmodules, you can automatically initialize the submodule with the --init option.
```

I would add it just to be safe.

**Comment by ricardo8c** [comment]
Hello Guys. I still have the VNC issue described in https://github.com/aind-containers/aind/issues/47
What I have to do to fix it? I don't now how to restart the VNC server...

Thanks

**Pull request merged**